framework/src/main/java/org/checkerframework/framework/flow/CFAbstractAnalysis.java

```diff
@@ -51,7 +51,7 @@
     protected final DependentTypesHelper dependentTypesHelper;
 
     /** A type factory that can provide static type annotations for AST Trees. */
-    protected final GenericAnnotatedTypeFactory<V, S, T, ? extends CFAbstractAnalysis<V, S, T>>
+    public final GenericAnnotatedTypeFactory<V, S, T, ? extends CFAbstractAnalysis<V, S, T>>
```

Comment:
Should this change be reverted?  You can call `getTypeFactory()` to access this field.